tardis/visualization/widgets/shell_info.py

```diff
@@ -4,7 +4,7 @@
     atomic_number2element_symbol,
     species_tuple_to_string,
 )
-from tardis.simulation import Simulation
+
```

Comment:
Was Simulation not being used in the module?

Reply:
I'm afraid not